src/main/java/com/researchspace/zenodo/model/ZenodoDeposition.java

```diff
@@ -0,0 +1,59 @@
+package com.researchspace.zenodo.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+/**
+ * A Zenodo Deposition, as returned by the API.
+ */
+@Data
+@NoArgsConstructor
```

Comment:
public non-args constructor is default in java, so if class just need that, then no need for annotation